test/controllers/topics_controller_test.rb

```diff
@@ -273,14 +273,41 @@ class TopicsControllerTest < ActionController::TestCase
     end
   end
 
+  test 'a browsing user should not be able to create a new topic when failing the recaptcha test' do
+    # Make sure recaptcha is set up
+    Recaptcha.configuration.skip_verify_env.delete("test")
```

Comment:
This seems to be the only way to enable the recaptcha verification in the test env. See https://github.com/ambethia/recaptcha#testing